scripts/bash/getDeployablePackages.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
-echo "admin-panel" "lesmis" "psss" "web-frontend" "admin-panel-server" "central-server" "entity-server" "lesmis-server" "meditrak-app-server" "psss-server" "report-server" "web-config-server"
+echo "admin-panel-server" "central-server" "entity-server" "lesmis-server" "meditrak-app-server" "psss-server" "report-server" "web-config-server" "admin-panel" "lesmis" "psss" "web-frontend"
```

Comment:
Just wanted to double check this works? I thought it was the `admin-panel-server` that depended upon `report-server`? 

```suggestion
echo "report-server" "admin-panel-server" "central-server" "entity-server" "lesmis-server" "meditrak-app-server" "psss-server" "web-config-server" "admin-panel" "lesmis" "psss" "web-frontend"
```

Alternatively we could flag the `report-server` as an internal dependency, as I did in https://github.com/beyondessential/tupaia/pull/3956 but that requires a few more code changes and is confusing in its own right...

Reply:
Oh you're right!